{{cookiecutter.project_slug}}/.gitlab-ci.yml

```diff
@@ -73,7 +79,7 @@ test_{{ cookiecutter.project_slug }}_install:
     - ls
     - python -c "import {{ cookiecutter.project_slug }}; print({{ cookiecutter.project_slug }})"
   only:
-    - master
+    - main
```

Comment:
So far, I have no package with a 'main' branch. Using 'main' and 'master' branches could be confusing. Should we rename all the 'master' branches?

Reply:
When we create a new GitLab repository, the default branch is now called `main`. We are just updating to the new way of GitLab creating the default branch ([info](https://about.gitlab.com/blog/2021/03/10/new-git-default-branch-name/)).

Reply:
Ok, it seems like the master branch is renamed in the whole Git ecosystem. So we should also rename it.